src/main/resources/config/liquibase/master.xml

```diff
@@ -156,6 +156,7 @@
     <include file="classpath:config/liquibase/changelog/20220210155100_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20220218140000_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20220303030303_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20220406142500_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
To maintain chronology, you should add your change at the end :)

Reply:
That must have come from merging `develop`… should I update the timestamp as well?